dist/main.css

```diff
@@ -0,0 +1,21 @@
+#page {
+  display: grid;
+  grid-template-columns: 8fr
```

Comment:
You forgot a semi-colon at the end of the line.

Reply:
Thank you, good catch.